v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Disaster Recovery
+summary: Learn how about CockrachDB disaster recovery capabilities and what to do if you encounter an issue.
+toc: true
+---
+
+While CockroachDB is built to be fault-tolerant and to recover automatically, sometimes disasters happen. A _disaster_ is any event that puts your cluster at risk, and usually means your cluster is experiencing [hardware failure](#hardware-failure), [data failure](#data-corruption), or has [compromised security keys](#compromised-security-keys). Having a disaster recovery plan enables you to recover quickly, while limiting the consequences.
```

Comment:
nit: Starting with `While` diminishes the value of our fault tolerance abilities. Personally prefer coming out strong with that message and highlighting that even with that there are all types of disasters to recover from.

Reply:
Good point. Edited.